src/dds/rs-dds-device-proxy.cpp

```diff
@@ -209,6 +209,28 @@ dds_device_proxy::dds_device_proxy( std::shared_ptr< context > ctx, std::shared_
             }
         } );  // End foreach_stream lambda
 
+    for( auto & sensor_info : sensor_name_to_info )
+    {
+        sensor_info.second.proxy->initialization_done( dev->device_info().product_id, dev->device_info().product_line );
+
+        // Set profile's ID based on the dds_stream's ID (index already set). Connect the profile to the extrinsics graph.
+        for( auto & profile : sensor_info.second.proxy->get_stream_profiles() )
+        {
+            sid_index type_and_index( profile->get_stream_type(), profile->get_stream_index());
+            
+            auto & streams = sensor_info.second.proxy->streams();
+            
+            sid_index sidx = type_and_index_to_dds_stream_sidx[type_and_index];
```

Comment:
This will create it if it does not exist... and I assume we'll then `continue`, but this error condition isn't caught

Reply:
Added debug log entry before continuing